### DISCUSSION TITLE
Arranger Mode Aggressive Blinking

### DISCUSSION THREAD

**Discussion by stirman** [SynthstromAudible/DelugeFirmware#518] [Ideas]
I spend a lot of time in Arranger mode and that "Song" button fast blinking non-stop is driving me crazy :)

Idea: slow down the rate that it blinks, or ideally, make it pulse slowly... fading on/off slowly. Can you adjust underling LED brightness?

**Comment by sapphire-arches** [comment]
The LEDs are unfortunately either full-on or full-off (so fading is not possible) The flashing frequency could be modified here: https://github.com/SynthstromAudible/DelugeFirmware/blob/55a61106ed54445cf716d327e5328782a74d1ecf/src/definitions_cxx.hpp#L259-L260 or a menu item added to control the same. Units are in milliseconds.

**Reply by stirman** [reply]
Interesting. I'm not a good enough engineer to start submitting PR's to the community firmware, but simply slowing down the blink seems easy enough, so maybe I'll give it a try. Thank for pointing me to the right spot in the code!

**Reply by sapphire-arches** [reply]
I think if we were to start messing with the blink frequency, it would be best to add a menu item -- somewhere under the global settings menu.
